python: can't open file '/cluster/home/callen/projects/mae/figure_eigenpatches.py': [Errno 2] No such file or directory
